_events/2015-09-25.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Music Education Hackathon"
+description: "Music Education Hackathon"
```

Comment:
One potential description: Talks, hacks, and workshops focusing on the intersection of music and education